configure.ac

```diff
@@ -327,13 +327,6 @@ AC_ARG_ENABLE(redhat,
 	]
 )
 
-AC_ARG_ENABLE(redhat-sysv,
-	[  --enable-redhat-sysv    obsoleted ],[
-	echo "ERROR: --enable-redhat-sysv is obsoleted. Use --enable-redhat."
-	exit 1
-	]
-)
```

Comment:
The `--enable-redhat` argument needs to be removed as well.

Reply:
Done